spa/src/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const singleSpa = "single-spa";
+export const react = "react";
```

Comment:
This is a nit, but I don't think that this provides value. If ever there were constants that were acceptable to inline, these are them.